### PR TITLE
Add release notes for 1.3.6 release

### PR DESCRIPTION
**Pull request by zelinh**
Signed-off-by: Zelin Hao <zelinhao@amazon.com>

### Description
Add release notes for 1.3.6

### Issues Resolved
Part of https://github.com/opensearch-project/opensearch-build/issues/2650

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-release-notes-1.3.6.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+* Updated Kotlin to 1.6.0 ([#1052](https://github.com/opensearch-project/observability/pull/1052))
+* Update Jackson to 2.13.4 ([#1066](https://github.com/opensearch-project/observability/pull/1066))
+
+### Opensearch Security
+* Scope updateVersion to only build.gradle ([#2121](https://github.com/opensearch-project/security/pull/2121))
+
+### Opensearch Security Dashboards Plugin
+* Fixed the tenant switching after timeout ([#1090](https://github.com/opensearch-project/security-dashboards-plugin/pull/1090))
+* Select tenant popup only appears when mutli-tenacy is enabled ([#965](https://github.com/opensearch-project/security-dashboards-plugin/pull/965))
+* Fix for Tenancy info getting lost on re-login in SAML Authentication flow ([#1125](https://github.com/opensearch-project/security-dashboards-plugin/pull/1125))
```
Comment (zelinh) [inline release-notes/opensearch-release-notes-1.3.6.md:44]: After synced up offline with @vinayak15, this commit hasn't been updated in the release notes for security dashboards yet but will do shortly.

File: release-notes/opensearch-release-notes-1.3.6.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* Bump version to 1.3.6 ([#473](https://github.com/opensearch-project/dashboards-reports/pull/473))
+
+### OpenSearch Ml Commons
+* Increment version to 1.3.6 and remove jackson force version ([#454](https://github.com/opensearch-project/ml-commons/pull/454))
+
+### OpenSearch Observability
+* Bump version to 1.3.6 ([#1059](https://github.com/opensearch-project/observability/pull/1059))
+
+### Opensearch Security
+* Updates jackson version to 2.13.4 ([#2128](https://github.com/opensearch-project/security/pull/2128))
```
Comment (bbarani) [inline release-notes/opensearch-release-notes-1.3.6.md:66]: Nit: Update vs updates

**Review by elfisher — APPROVED** [review]
LGTM.

**Comment by krisfreedain** [comment]
Seems good to me (aside from @bbarani comment above)

**Review by bbarani — APPROVED** [review]

**Pull request merged**